pkg/js/validator/reporters.ts

```diff
@@ -0,0 +1,83 @@
+import { ModelValidationSingleError, ValidationError } from "../errors";
+import { Keyword, ReservedKeywords } from "./Keywords";
+
+
+interface ReporterOpts extends BaseReporterOpts {
```

Comment:
These are not needed, you are returning errors that should have all the information - it is not the language's job to build the custom reporter for VS Code/Monaco, that will have to be done elsewhere